### PR TITLE
DDBTEAM-538: Switched to agency-auth-bundle

### PR DESCRIPTION
**Pull request by turegjorup**

### CODE REVIEW

File: config/packages/security.yaml [hunk 1]
```diff
@@ -12,10 +12,13 @@ security:
...
+                    - DanskernesDigitaleBibliotek\AgencyAuthBundle\Security\TokenAuthenticator
         main:
             anonymous: true
 
     access_control:
         # Allows accessing the Swagger UI
         - { path: '^%app.path.prefix%/docs', roles: IS_AUTHENTICATED_ANONYMOUSLY }
+
+    role_hierarchy:
+        ROLE_OPENPLATFORM_AGENCY: [ROLE_COVER_READ]
```
Comment (cableman) [inline config/packages/security.yaml:24]: Missing new line :-)

**Comment by cableman** [comment]
In the other 2 PR's there where some old config in yml files that had been removed. But not in this one?

**Comment by cableman** [comment]
Was thinking about these 3.

```
$bindOpenPlatformId: '%env(OPENPLATFORM_ID)%'
$bindOpenPlatformSecret: '%env(OPENPLATFORM_SECRET)%'
$bindOpenPlatformIntrospectionUrl: '%env(OPENPLATFORM_INTROSPECTION_URL)%'
```

**Comment by turegjorup** [comment]
> Was thinking about these 3.
> 
> ```
> $bindOpenPlatformId: '%env(OPENPLATFORM_ID)%'
> $bindOpenPlatformSecret: '%env(OPENPLATFORM_SECRET)%'
> $bindOpenPlatformIntrospectionUrl: '%env(OPENPLATFORM_INTROSPECTION_URL)%'
> ```

Good catch

**Pull request merged**